everviz/plugins/lineplot/lineplot.py

```diff
@@ -0,0 +1,102 @@
+
+import plotly.graph_objs as go
+import plotly.express as px
+from pathlib import Path
+from uuid import uuid4
+from dash.dependencies import Output, Input
+from webviz_config import WebvizPluginABC
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+from everviz.data.load_csv.get_data import get_data
+
+
+class Lineplot(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Lineplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.filter_box_id = f"dropdown-{uuid4()}"
+        self.radio_id = f"dropdown-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+        ASSETS_DIR = Path(
+            pkg_resources.resource_filename("everviz", os.path.join("assets"))
+        )
+
+        WEBVIZ_ASSETS.add(ASSETS_DIR / "axis_customization.css")
+
+    def add_webvizstore(self):
+        return [(get_data, [{"data_path": self.data_path}])]
+
+    @property
+    def layout(self):
+        df = get_data(self.data_path)
+        radio_options = ["Statistics", "Data"]
+        return html.Div(
+            [
+                html.H1(children=self.title, style={"textAlign": "center"}),
+                html.Div(
+                    [
+                        html.Div(
+                            [
+                                dcc.RadioItems(
+                                    id=self.radio_id,
+                                    options=[
+                                        {"label": i, "value": i} for i in radio_options
+                                    ],
+                                    value=radio_options[0],
+                                    labelStyle={"display": "inline-block"},
+                                ),
+                            ],
+                            style={
+                                "width": "29%",
+                                "display": "inline-block",
+                                "vertical-align": "top",
+                            },
+                        ),
+                        html.Div(
+                            [dcc.Graph(id=self.graph_id)],
+                            style={"width": "69%", "display": "inline-block"},
+                        ),
+                    ]
+                ),
+            ]
+        )
+
+    def set_callbacks(self, app):
+        @app.callback(
+            Output(self.graph_id, "figure"), [Input(self.radio_id, "value"),],
+        )
+        def update_graph(graph_type):
+            df = get_data(self.data_path)
+            traces = []
+            if graph_type == "Statistics":
+                for label in ["P10", "P50", "P90"]:
+                    fill = "tonexty" if label in ["P90", "P50"] else None
+                    traces.append(
+                        go.Scatter(
+                            y=df[label],
+                            x=df["batch"],
+                            name=label,
```

Comment:
Maybe it is good to label at least the P10 and P90 lines by adding a P10/P90 suffix to the name? Also I think this way you get all three in the legend, which is maybe too much.

Reply:
Will do 👍